src/main/kotlin/mate/academy/supplier/FigureSupplier.kt

```diff
@@ -0,0 +1,65 @@
+package mate.academy.supplier
+
+import mate.academy.model.model.Circle
+import mate.academy.model.model.Figure
+import mate.academy.model.model.IsoscelesTrapezoid
+import mate.academy.model.model.Rectangle
+import mate.academy.model.model.RightTriangle
+import mate.academy.model.model.Square
+import kotlin.random.Random
+
+private const val MAX_SIZE: Double = 10.0
+private const val MIX_SIZE: Double = 1.0
+private const val DEFAULT_SIZE: Double = 10.0
+private const val FIGURES_NUMBER = 5
+private const val DEFAULT_COLOR = "WHITE"
+private const val SQUARE_INDEX = 0
+private const val RECTANGLE_INDEX = 1
+private const val CIRCLE_INDEX = 2
+private const val RIGHT_TRIANGLE_INDEX = 3
+
+
+class FigureSupplier {
+
+    private val colorSupplier = ColorSupplier()
+    private val random = Random
```

Comment:
Please check whether kotlins Random is required to be extracted as separate value that keeps instance.
Maybe it is smth like static or ...  who khows :)